src/main/java/org/example/homework3/MinAndMax.java

```diff
@@ -0,0 +1,15 @@
+package org.example.homework3;
+
+public class MinAndMax {
+    public static void main(String[] args) {
+        int num = 11;
+        int num1 = 5;
```

Comment:
`num` and `num1` are not very clear names. for me, it is more confusing. 
For this task it is not critical, but in future try to use more informative names (e.g. `firstNumber`, `secondNumber`)